Master-OB-OpenAPI.json

```diff
@@ -6518,7 +6542,9 @@
             "x-ob-item-type": "DateItemType",
             "x-ob-item-type-group": "",
             "x-ob-usage-tips": "",
-            "x-ob-sample-value": {}
+            "x-ob-sample-value": {
+              "Value": "2022-12-28T19:49:26Z"
```

Comment:
Splitting hairs here: the element is CommentDate, of DateItemType, the example is a full datetime value. In other instances, e.g., CertificationDate, the example is only a date string. Should examples make the distinction between date and datetime strings?